src/mbgl/renderer/paint_property_binder.hpp

```diff
@@ -302,11 +327,8 @@ class SourceFunctionPaintPropertyBinder final : public PaintPropertyBinder<T, T,
         }
     }
 
-    void upload([[maybe_unused]] gfx::UploadPass& uploadPass) override {
 #if MLN_LEGACY_RENDERER
-        vertexBuffer = uploadPass.createVertexBuffer(std::move(vertexVector));
-#endif // MLN_LEGACY_RENDERER
-    }
+    void upload(gfx::UploadPass& uploadPass) override { vertexBuffer = uploadPass.createVertexBuffer(vertexVector); }
 
     std::tuple<std::optional<gfx::AttributeBinding>> attributeBinding(
```

Comment:
That's after 1e20bb6?  I moved `attributeBinding` into the `MLN_LEGACY_RENDERER` (since it interacts with the buffers) and was able to build all three variants cleanly on Linux using gcc-12.1.

Reply:
You're right, I did another look and it builds fine - guessing git failed to fast-forward.